crowdfunding_compassion/controllers/project_controller.py

```diff
@@ -22,8 +22,50 @@ def project_page(self, project, **kwargs):
             self._prepare_project_values(project.sudo(), **kwargs),
         )
 
-    # TODO: test when we can create data for a project
+    @route(["/participant/<int:project_id>/<int:participant_id>"],
```

Comment:
The participant is unique for each project, because if the same person registers for multiple projects it will create a new participant record. Therefore I propose to only use participant in the route and use model for more user friendly route (names instead of ids only).
```suggestion
    @route(["/participant/<model('crowdfunding.participant'):participant>/"],
```